React/Views/RCTScrollView.m

```diff
@@ -946,6 +955,9 @@ - (type)getter                                            \
 #if !TARGET_OS_TV
 RCT_SET_AND_PRESERVE_OFFSET(setPagingEnabled, isPagingEnabled, BOOL)
 RCT_SET_AND_PRESERVE_OFFSET(setScrollsToTop, scrollsToTop, BOOL)
+#else
+RCT_EMPTY_BOOL_GETTER_SETTER(setPagingEnabled, isPagingEnabled)
```

Comment:
Why is this necessary? I believe RN wil automatically ignore attributes that are not exported by a view manager (which is how we OS-specific attributes)